.github/workflows/syndicate.yml

```
@@ -17,15 +17,20 @@ on:
 permissions:
   contents: write
+  pull-requests: write
 
 jobs:
   syndicate:
-    if: github.actor != 'github-actions[bot]'
+    # Skip when this run was triggered by the syndicate workflow's own merge to
+    # main (commit message comes from the PR title set by the composite action).
+    # The `|| ''` fallback keeps workflow_dispatch runs working — head_commit
+    # is undefined for that event.
+    if: ${{ !startsWith(github.event.head_commit.message || '', 'chore: update syndication data') }}
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)